### PR TITLE
Typo fix: recognise/does not keeping

### PR DESCRIPTION
**Pull request by AdamDang**
Two typos:

Line 10: does not keeping->does not keep
Line 231: recognise->recognize

### CODE REVIEW

File: contributors/design-proposals/cli/preserve-order-in-strategic-merge-patch.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ If an old kubectl sends a old patch to a new server,
 the server will honor the order in the list as mentioned above.
 The behavior is a little different from before but is not a breaking change.
 
-If a new kubectl sends a new patch to an old server, the server doesn't recognise the parallel list and will drop it.
+If a new kubectl sends a new patch to an old server, the server doesn't recognize the parallel list and will drop it.
```
Comment (cblecker) [inline contributors/design-proposals/cli/preserve-order-in-strategic-merge-patch.md]: Please remove this change. "recognise" is the British English spelling of the term, and either is acceptable in our documentation (at the author's discretion)

Comment (AdamDang) [reply contributors/design-proposals/cli/preserve-order-in-strategic-merge-patch.md]: OK, It's removed now.
Thanks!

**Comment by nikhita** [comment]
@AdamDang can you squash the commits? This looks good to merge! :rocket:

**Comment by cblecker** [comment]
/lifecycle rotten

@AdamDang If you're able to squash, we'll merge this. Otherwise I'm going to queue this to close in 30d.

**Pull request closed without merging**